workflow/scripts/process/process_power_1_powerplants.py

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
As with the intersect scripts, let's remove the filename numbers unless there are really good reasons to keep them.